<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <rosparam param="/marker_publisher/marker_list"> [11, 12, 13, 15] </rosparam>

    <node pkg="rosbot_bringup" type="realworld.py" name="realworld" output="screen">
    </node>

    <node pkg="aruco_ros" type="marker_publisher" name="aruco_marker_publisher" output="screen">
        <remap from="/image" to="/camera/rgb/image_raw"/>
    </node>

</launch>

